index.php

```diff
@@ -0,0 +1,50 @@
+<?php
+    $full_name="";
+    if (isset($_SERVER["REQUEST_METHOD"])=="POST") {
+        if (isset($_POST['submit']))
+        {
+            $first_name = htmlspecialchars($_POST['first_name']);
+            $last_name = $_POST['last_name'];
```

Comment:
Why are we using the `htmlspecialchars` function in first_name and not in the last_name?